artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ClusterControl.java

```diff
@@ -64,6 +62,10 @@ public ClusterControl(ClientSessionFactoryInternal sessionFactory, ActiveMQServe
       this.clusterPassword = server.getConfiguration().getClusterPassword();
    }
 
+   public Optional<Channel> getClusterChannel() {
```

Comment:
@michaelandrepearce @brusdev Not proud to have "exposed" the lower level `Channel` here, but it seems the simplest way to move away he many  `Quorum*` deps away from here